db/migrate/20150120094831_create_pins.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :pininterest_id
       t.string :title
       t.string :optional_info
```

Comment:
There is some description with some of the pins, which I want to scrape but it will be stored likely with an association join table later, when we build the associations. So it will be kept with the pin originally, but not actually displayed from there. Sounds a bit complex, but I think of it as a temporary storage place until we set up the other table